docs/butterfly/pistonball.md

```diff
@@ -17,9 +17,9 @@ agent-labels: "agents= ['piston_0', 'piston_1', ..., 'piston_19']"
 
 
 
-This is a simple physics based cooperative game where the goal is to move the ball to the left wall of the game border by activating any of the twenty vertically moving pistons. Each piston agents observation is an RGB image of the area the, the two pistons next to them can move in (or the wall for pistons against the wall). Every piston can be acted on in any given time, the action space in discrete mode is 0 for down, 1 for staying still, and 2 for up. In continuous mode, the value is proportional to the amount the pistons are raised or lowered by.
+This is a simple physics based cooperative game where the goal is to move the ball to the left wall of the game border by activating the vertically moving pistons. Each piston agent's observation is an RGB image of the two pistons next to them and the space above them (or the wall for pistons against the wall). Every piston can be acted on in any given time. The action space in discrete mode is 0 to move down, 1 to stay still, and 2 to move up. In continuous mode, the value in the range [-1, 1] is proportional to the amount that the pistons are raised or lowered by.
```

Comment:
Proportional how?

Reply:
In a way that doesn't make a lot of sense. In discrete space, you choose between (1) moving up 4 pixels, (0) nowhere, or (-1) down 4 pixels. In continuous space, we scale actions by 4 so that the inputs 1 and -1 do the same exact thing in discrete and continuous space. Basically the movement is 4x the action

Reply:
Say that the base unit for continuous is 4 pixels then?